proposals/lip-0063.md

```diff
@@ -332,24 +342,29 @@ In the following, let `accountsState` be the key-value store of the accounts sta
 Let `genesisLegacyStoreSchema` be as defined in [LIP 0050](https://github.com/LiskHQ/lips/blob/aebeecc40447e4229f14fc641fb11527270dfbb5/proposals/lip-0050.md#genesis-state-initialization). The `assets` entry for the legacy module is added by the logic defined in the function `addLegacyModuleEntry` in the following pseudo code:
 
 ```python
-addLegacyModuleEntry():
+def addLegacyModuleEntry():
     legacyObj = {}
     legacyObj.accounts = []
+    legacyReserveAmount = 0
     for every account in legacyAccounts:
         userObj = {}
         userObj.address = account.address
         userObj.balance = account.token.balance
         legacyObj.accounts.append(userObj)
+        legacyReserveAmount += account.token.balance
     data = serialization of legacyObj using genesisLegacyStoreSchema
     append {"moduleID": MODULE_ID_LEGACY, "data": data} to b.assets
+    # mint all the tokens for legacy accounts here
+    Token.mint(ADDRESS_LEGACY_RESERVE, TOKEN_ID_LSK_MAINCHAIN, legacyReserveAmount)
+    Token.lock(ADDRESS_LEGACY_RESERVE, MODULE_ID_LEGACY, TOKEN_ID_LSK_MAINCHAIN, legacyReserveAmount)
 ```
```

Comment:
I think this logic is not really following the migration paradigm.
The migration should not mint + lock tokens. This amount should be  added to the `assets` entry for the token module under the correct address.